utils/tinystr/Cargo.toml

```diff
@@ -45,7 +43,7 @@ rand = { version = "0.8.5", features = ["small_rng"] }
 default = ["alloc"]
 alloc = []
 bench = []
-zerovec = ["dep:zerovec", "dep:zerovec_07"]
+zerovec = ["dep:zerovec"]
```

Comment:
Suggestion (very optional; can do later): remove this feature now that it is automatic again